core/state_transition.go

```diff
@@ -32,9 +32,10 @@ import (
 )
 
 var (
-	errInsufficientBalanceForGas    = errors.New("insufficient balance to pay for gas")
-	errNonWhitelistedGasCurrency    = errors.New("non-whitelisted gas currency address")
-	errGasPriceDoesNotExceedMinimum = errors.New("gasprice does not exceed gas price minimum")
+	ErrGasPriceDoesNotExceedMinimum = errors.New("gasprice does not exceed gas price minimum")
```

Comment:
Nit: *Capitalize* the first letters of each sentence.

Reply:
FWIW, all errors in geth start with a lower case, so for the sake of consistency, I am going to keep it lowercase.